src/app/tests/crud/test_crud_project.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.models.project import Project
+from app.crud.crud_project import project
+from app.schemas.project import ProjectUpdate
+from app.tests.utils.models import create_project
+
+pytestmark = pytest.mark.asyncio
+
+
+async def test_crud_project_create(database: AsyncSession):
+    project_obj = await create_project(db=database)
+    assert project_obj is not None
+    assert isinstance(project_obj, Project)
```

Comment:
Please also check the that the object has arrived in the database. (And not just that I return the request body and still get a pass :wink:)

Reply:
Addressed